.github/workflows/ghstack_land.yaml

```diff
@@ -77,12 +78,12 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.LAND_TOKEN }}
 
     - name: Create URL to the run output
-      if: ${{ failure() }}
+      if: ${{ failure() && steps.landPR.outcome == 'failure' }}
```

Comment:
you added an ID 'landStack' and this step reference 'landPR' ... were those supposed to match?

Reply:
Good catch, thank you. Fixed now.